### PR TITLE
Gnss antenna

### PR DESCRIPTION
**Pull request by frankfu03**

### CODE REVIEW

File: src/client/app/datetime-picker/datetime-picker.component.ts [hunk 1]
```diff
@@ -39,6 +39,9 @@ export class DatetimePickerComponent implements OnInit, DoCheck {
    * Initialize relevant variables when the directive is instantiated
    */
```
Comment (oehm-smith) [inline src/client/app/datetime-picker/datetime-picker.component.ts]: You'll need to rebase and rework this as Heya's recent work (merged) moved the DateTimePicker.

File: src/client/app/gnss-antenna/gnss-antenna-item.component.html [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+    </span>
+    <span class="pull-right">
+      <button class="btn btn-danger btn-rm"
+              title="Remove the antenna"
+              (click)="removeItem(index)">
+        <i class="glyphicon glyphicon-remove"></i>
+        <span class="hidden-xs">Remove</span>
+      </button>
+    </span>
+    <!--<span *ngIf="index == 1 && total >= 3" class="pull-right">
```
Comment (oehm-smith) [inline src/client/app/gnss-antenna/gnss-antenna-item.component.html]: Lets not keep commented out code.

File: src/client/app/gnss-antenna/gnss-antenna-item.component.html [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+                           [name]=" 'Date Removed' "
+                           [required]="index != 0"></datetime-picker>
+        </div>
+      </div>
+      <div class="form-group">
+        <label class="col-md-4 col-sm-4 col-xs-5 col-xxs-12 control-label">Notes</label>
+        <div class="col-md-8 col-sm-8 col-xs-7 col-xxs-12">
+          <textarea maxlength="2000" rows="2" class="form-textarea"
+                    [(ngModel)]="antenna.notes"
+                    name="antennaNotes_{{index}}"></textarea>
```
Comment (oehm-smith) [inline src/client/app/gnss-antenna/gnss-antenna-item.component.html]: Could you get this validation working before we implement too many of these?  (In a separate branch after this is merged).

File: src/client/app/site-info/site-info.component.html [hunk 4]
```diff
@@ -123,237 +123,24 @@
...
-                                name="antennaNotes_{{i}}"></textarea>
-                      </div>
-                    </div>
-                  </div>
-                </div>
-              </div>
-            </div>
-          </div>
-        </div>
+        <gnss-receiever [status]="status"
```
Comment (oehm-smith) [inline src/client/app/site-info/site-info.component.html]: It looks like you fixed up the receivers on this same branch.  Could you move this work to a separate branch and PR.

File: src/client/app/site-info/site-info.component.html [hunk 5]
```diff
@@ -123,237 +123,24 @@
...
+                        [gnssReceivers]="receivers"
+                        [siteLogModel]="siteLogModel"
+                        [siteLogOrigin]="siteLogOrigin"></gnss-receiever>
+        <gnss-antenna-group [siteLogModel]="siteLogModel"
+                            [originalSiteLogModel]="siteLogOrigin"></gnss-antenna-group>
         <gnss-surveyed-local-tie [status]="status"
-			[surveyedLocalTies]="surveyedLocalTies" [siteLogModel]="siteLogModel"
-			[siteLogOrigin]="siteLogOrigin"></gnss-surveyed-local-tie>
+                                 [surveyedLocalTies]="surveyedLocalTies"
```
Comment (oehm-smith) [inline src/client/app/site-info/site-info.component.html]: Similarly to above comment about too much work in this branch / pr, lets move these other mods out to their own also.

File: src/client/app/site-info/site-info.module.ts [hunk 6]
```diff
@@ -18,7 +19,8 @@ import { WaterVaporSensorModule } from '../water-vapor-sensor/water-vapor-sensor
     CommonModule,
     SharedModule,
     ResponsiblePartyModule,
-    GnssReceiverInfoModule,
+    GnssReceiverModule,
```
Comment (oehm-smith) [inline src/client/app/site-info/site-info.module.ts]: Another bit of code that needs to be moved to its own branch / PR.

File: src/client/app/datetime-picker/datetime-picker.component.ts (hunk 1 above)
Comment (frankfu03) [inline src/client/app/datetime-picker/datetime-picker.component.ts]: We may do it in another branch.

File: src/client/app/gnss-antenna/gnss-antenna-item.component.html (hunk 2 above)
Comment (frankfu03) [inline src/client/app/gnss-antenna/gnss-antenna-item.component.html]: The commented-out code have been removed.

File: src/client/app/datetime-picker/datetime-picker.component.ts (hunk 1 above)
Comment (oehm-smith) [inline src/client/app/datetime-picker/datetime-picker.component.ts]: Yes thanks

File: src/client/app/site-info/site-info.component.ts [hunk 7]
```diff
@@ -427,7 +336,7 @@ export class SiteInfoComponent implements OnInit, OnDestroy {
 
   /**
    * Set current and previous antennas, and their show/hide flags
-   */
```
Comment (oehm-smith) [inline src/client/app/site-info/site-info.component.ts]: Can we get rid of this?

File: src/client/app/gnss-antenna/gnss-antenna-item.component.html (hunk 3 above)
Comment (frankfu03) [inline src/client/app/gnss-antenna/gnss-antenna-item.component.html]: Yeah, will do it in a separate branch.

File: src/client/app/site-info/site-info.component.html (hunk 4 above)
Comment (frankfu03) [inline src/client/app/site-info/site-info.component.html]: Rolled back.

File: src/client/app/site-info/site-info.component.ts (hunk 7 above)
Comment (frankfu03) [inline src/client/app/site-info/site-info.component.ts]: All commented-out code has been removed.

**Pull request merged**